components/Onboarding/Sections/Profile.js

```diff
@@ -0,0 +1,193 @@
+import React, { Component } from 'react'
+import { Query, Mutation } from 'react-apollo'
+import gql from 'graphql-tag'
+import { css, merge } from 'glamor'
+
+import withMe from '../../../lib/apollo/withMe'
+import { TESTIMONIAL_IMAGE_SIZE } from '../../constants'
+
+import Portrait from '../../Profile/Portrait'
+import Statement from '../../Profile/Statement'
+import { ListedCheckbox, PublicCheckbox } from '../../Profile/Settings'
+import { mutation } from '../../Profile/Edit'
+import UsernameField from '../../Profile/UsernameField'
+
+import {
+  Interaction,
+  Button,
+  FieldSet,
+  mediaQueries
+} from '@project-r/styleguide'
+
+import Section from '../Section'
+
+const { P } = Interaction
+
+const PORTRAIT_SIZE_M = TESTIMONIAL_IMAGE_SIZE
+const PORTRAIT_SIZE_S = TESTIMONIAL_IMAGE_SIZE * 0.75
+
+const styles = {
+  p: css({
+    marginBottom: 20
+  }),
+  field: css({
+    marginTop: 10,
+    marginBottom: 10
+  }),
+  checkbox: css({
+    marginTop: 40,
+    marginBottom: 40
+  }),
+  portrait: css({
+    width: PORTRAIT_SIZE_S,
+    height: PORTRAIT_SIZE_S,
+    [mediaQueries.mUp]: {
+      width: PORTRAIT_SIZE_M,
+      height: PORTRAIT_SIZE_M
+    }
+  }),
+  actions: css({
+    marginBottom: 20,
+    display: 'flex',
+    flexWrap: 'wrap',
+    position: 'relative',
+    '& > button': {
+      flexGrow: 1,
+      margin: '5px 15px 0 0',
+      minWidth: '120px',
+      [mediaQueries.mUp]: {
+        flexGrow: 0,
+        margin: '5px 15px 0 0',
+        minWidth: '160px'
+      }
+    }
+  })
+}
+
+const CONDENSED_PROFILE = gql`
+  query condensedProfile {
+    me {
+      id
+      firstName
+      lastName
+      portrait
+      statement
+      username
+      isAdminUnlisted
+      isListed
+      isEligibleForProfile
+      hasPublicProfile
+    }
+  }
+`
+
+class Profile extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      values: {},
+      errors: {},
+      dirty: {}
+    }
+
+    this.onChange = fields => {
+      this.setState(FieldSet.utils.mergeFields(fields))
+    }
+
+    this.onCompleted = () => {
+      props.onContinue.apply(null, [props])
+    }
+  }
+
+  render () {
+    const { onContinue } = this.props
+    const { values: stateValues, errors, dirty } = this.state
+
+    return (
+      <Query query={CONDENSED_PROFILE}>
+        {({ loading, error, data: { me: queryValues } = {} }) => {
+          const mergedValues = Object.assign({}, queryValues, stateValues)
```

Comment:
```suggestion
          const mergedValues = Object.assign({}, queryValues, {portrait: undefined}, stateValues)
```
and rm the change in `components/Profile/Portrait.js`